### PR TITLE
Remove data

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

**Comment by martinRenou** [comment]
Tests coming

File: ipysheet/pandas_loader.py [hunk 1]
```diff
@@ -95,17 +128,13 @@ def _extract_column(data, idx):
...
-    print(type)
+
     if type == 'date':
         d = pd.to_datetime(arr)
 
         return np.array(d, dtype='M')
-    elif type == 'checkbox':
-        return np.array(arr, dtype=np.bool)
-    elif type == 'numeric':
-        return np.array(arr, dtype='f')
```
Comment (martinRenou) [inline ipysheet/pandas_loader.py]: Those were not needed, NumPy can infer the dtype

**Comment by martinRenou** [comment]
@maartenbreddels Ready :D

**Comment by maartenbreddels** [comment]
I'll review and see if I can break it! ;)

**Comment by martinRenou** [comment]
It is unbreakable you know, I never write bugs, I don't really see the point of doing it ;)

**Pull request merged**